frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostsTable.tsx

```diff
@@ -103,6 +111,7 @@ const HostsTableRow: React.FC<HostsTableRowProps> = ({
         <BareMetalHostRole machine={machine} node={node} />
       </TableData>
       <TableData className={tableColumnClasses.address}>{address}</TableData>
+      <TableData className={tableColumnClasses.serialNumber}>{serialNumber}</TableData>
```

Comment:
```suggestion
      <TableData className={tableColumnClasses.serialNumber}>{serialNumber || DASH}</TableData>
```

Reply:
done, also added the `DASH` to `address`